lib/config.js

```diff
@@ -93,7 +93,7 @@ const configBaseSchema = v.object({
     nameCleaner: v.function.default(path => path.replace(/^app\//, ''))
   }).default({}),
 
-  notifications: v.bool.default(true),
+  notifications: v.either(v.bool.default(true), v.array(v.string).default([])).default(true),
```

Comment:
We don't need the nested defaults here as they won't be taken into account —only top-level one will be (`true`)

Please change to 

```javascript
v.either(v.bool, v.array(v.string)).default(true)
```